x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const SHOW_HOSTS_RISK_SCORE = i18n.translate(
+  'xpack.securitySolution.riskScore.riskScorePreview.hosts.show',
+  {
+    defaultMessage: 'Show hosts',
+  }
+);
+
+export const HIDE_HOSTS_RISK_SCORE = i18n.translate(
+  'xpack.securitySolution.riskScore.riskScorePreview.hosts.hide',
+  {
+    defaultMessage: 'Hide hosts',
```

Comment:
Hey @rylnd - Using verbs was deliberate. I'll try to elaborate, but obviously open to discussion as I'm still relatively new to Elastic and have not learned the research / design choices behind every component!

In my previous work in GOVUK, all the user research we did indicated that cognitive load and uncertainty was reduced when verbs were added as it gave the user confidence they knew what to expect when they clicked it. It tends to be best practice now on things like accordions, you can see it in the GOVUK Design System and even GitHub do it in this very thread, for example: "Hide all checks" and "Show resolved". 

From an accessibility point of view, it also means that the accessible name in the accessibility tree is less ambiguous when using something like a screen reader or voice controller. For example, the link to the explore pages are just labelled 'hosts' and `users`. So you'd have a potential situation where there are two interactions labelled identically but have completely different outcomes / expectations. 

If you're dead set on removing the verbs, we should at the very least use hidden context or alter the aria-labels to still include verbs or states. 

Hope this helps. C

Reply:
@craig-abbott I think my reaction centers around "Hide [noun]" being the default title when viewing the screen:

* "Hide" is a little tricky because it can at first glance look like an adjective as well as a verb; "Hide all [noun]" disambiguates this somewhat 
* I haven't personally seen any examples like this of an accordion being _open_ by default and showing a verb. It would be the equivalent of [this form](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/) displaying "Hide Personal Information" instead of just "Personal Information"

Not a hill I want to die on, though. I find it weird, but if I'm the only one, carry on!

Reply:
I felt the same way as you @rylnd but also not my area of expertise.